vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -196,6 +196,21 @@ + (void)setEnvironment:(int)environment
   ads::_environment = static_cast<ads::Environment>(environment);
 }
 
++ (void)sysInfo:(BATBraveAdsSysInfo *)sysInfo
```

Comment:
where does BATBraveAdsSysInfo come from and why are we using a different type/name here?

Reply:
Agreed will fix the g_ for globals, just kept consistent from legacy.

Reply:
We needed to namespace as we had conflicts with iOS/ledger. This needs further investigation and improving

Reply:
we shouldn't have to generate any classes for this because we can expose the mojo headers cc @kylehickinson

Reply:
can we raise a new issue to investigate/resolve this, as the issue is urgent? Thanks

Reply:
not urgent, but it would certainly simplify things to have one data structure class/name everywhere

Reply:
Fixed, thanks